### DISCUSSION TITLE
Is it possibile take custom label from nodes?

### DISCUSSION THREAD

**Discussion by lucky-sideburn** [open-telemetry/opentelemetry-collector#6850] [General]
Hi,

We are trying to replace the Datadog Agent with Otel Collector for filtering metrics that are importants for us. 

It works great but we are not able to add to the metrics custom labels related to the source worker nodes.

Can you please help us? Is it possible do it with OTEL?

I add this link https://docs.datadoghq.com/containers/kubernetes/tag/?tab=containerizedagent#node-labels-as-tags for explain better what we want to do.

Thanks
Eugenio

**Comment by jpkrohling** [comment]
If you are using the contrib distribution, you should take a look at this processor: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor

**Comment by lucky-sideburn** [comment]
Hi @jpkrohling thank you very much.

I will try it
